packages/react/src/components/Flex/FlexItem.tsx

```diff
@@ -49,9 +50,12 @@ class FlexItem extends UIComponent<ReactProps<FlexItemProps>> {
 
   static propTypes = {
     ...commonPropTypes.createCommon({
+      children: false,
```

Comment:
nit: `children` argument in createCommon can be boolean or string. Would it make sense to pass direct value (like you do on line 57) as well (or instead of the string)?

Reply:
That's an interesting question. I can add as a separate PR options for passing direct value for the children and content options.